CDP4-SDK-Samples/SampleApp.cs

```diff
@@ -0,0 +1,506 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SampleApp.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4.SDK.Samples
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Text;
+    using System.Threading.Tasks;
+    using CDP4Common.EngineeringModelData;
+    using CDP4Common.SiteDirectoryData;
+    using CDP4Common.Types;
+    using CDP4Dal;
+    using CDP4Dal.DAL;
+    using CDP4Dal.Operations;
+    using CDP4ServicesDal;
+    using NLog;
+    using NLog.Config;
+    using NLog.Targets;
+
+    public class SampleApp
+    {
+        private static ISession _session;
+        private static Credentials _credentials;
+        private static bool _isRunning = true;
+        private static Uri _uri;
+
+        // Commands
+        private const string Open = "open";
+        private const string Refresh = "refresh";
+        private const string Reload = "reload";
+        private const string Close = "close";
+        private const string Restore = "restore";
+        private const string GetIteration = "get_iteration";
+        private const string PostPerson = "post_person";
+        private const string PostParameter = "post_parameter";
+        private const string PostPfsl = "post_pfsl";
+        private const string PostPfslReorder = "post_pfsl_reorder";
+        private const string RemoveParameter = "remove_parameter";
+
+        static void Main(string[] args)
```

Comment:
Move all business logic to another class to have all these fields and methods static. And just instantiate that new class from here.